extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -129,30 +126,21 @@ export class FileConfigPage extends ImportPage {
 		return true;
 	}
 
-	public setupNavigationValidator() {
```

Comment:
use navigation validator is the correct way, not using this doesn't seem correct to me, maybe you could explain a bit more about why this is removed?

Reply:
Navigation validator should only be used for validations that can't be done by the components themselves (it's better to have the button greyed out if things on the page are invalid instead of waiting until the user clicks next to run the validation which is what navigation validator does) - in this case a component that's still loading is considered invalid (which is the only thing this validator is checking) so this validator was unnecessary.